.github/workflows/docker-latest.yml

```diff
@@ -0,0 +1,48 @@
+name: Publish Docker image for latest release
+
+on:
+  push:
+    tags:
+      - release
+
+env:
+  TEST_TAG: zeek:latest
```

Comment:
This tag isn't just for testing but is actually the one the image will be labeled with in the end, right? If it's not just for testing, a name like `DOCKER_IMAGE_TAG` might be better.

Reply:
No, this is the tag of an image created locally on the machine so we can run the container tests on it. For publishing we run the build action once again later (which reuses the cached contents), but specify different tags (e.g., `zeekurity/zeek:latest`).